lib/3scale_toolbox/commands/remote_command/remote_remove.rb

```diff
@@ -25,9 +28,8 @@ def run
         private
 
         def remove_remote(remote_name)
-          config.update(:remotes) do |remotes|
-            remotes = {} if remotes.nil?
-            remotes.tap do |r|
+          update_remotes do |rmts|
+            rmts.tap do |r|
```

Comment:
Why `rmts` over `remotes` ?

Reply:
trying not to shadow `remotes` method from `Remotes` module

Reply:
Is there some rubocop check for that? I don't see why. Shadowing a method by a local variable is pretty common in Ruby.

Reply:
nope, just common sense :)